pixorama_flutter/lib/src/connection_display.dart

```diff
@@ -30,8 +30,7 @@ class ConnectionDisplay extends StatelessWidget {
         connectionStateColor = Colors.red;
         break;
       case StreamingConnectionStatus.waitingToRetry:
-        connectionStateDescription =
-            'Retrying in ${connectionState.retryInSeconds} seconds';
+        connectionStateDescription = 'Retrying in ${connectionState.retryInSeconds} seconds';
```

Comment:
Why has this line changed? Are you using `dart format`? (Best practice is to set up VS Code so that it always applies `dart format` when you save a file.)

Reply:
I'm using dart format, once i save it formats to this